docker/docker-compose.test.yml

```diff
@@ -22,7 +22,6 @@ services:
     build:
       context: ..
       dockerfile: Dockerfile
-    command: py.test
```

Comment:
Command should be there if you want to use the same Dockerfile. Otherwise it will run whatever command is used in that Dockerfile, which I think is not `py.test`.